wfsim/strax_interface.py

```diff
@@ -632,6 +632,7 @@ def set_timing(self,):
             self.instructions_epix = self.instructions_epix[~extra_long]
             log.warning('Found and removing %d epix instructions later than maximum time %d'
                         % (extra_long.sum(), max_time))
+            self.instrutions_epix = np.sort(self.instructions_epix, order='time')
```

Comment:
We need to add the same for the nVeto target, don't we?

Reply:
But there's the sorting done in the https://github.com/XENONnT/WFSim/blob/master/wfsim/core.py#L1163-L1164 anyway. Sorting here is not necessary.

Reply:
Okay, this might be the reason for the misunderstanding with @petergaemers that we thought we had resolved this morning. Then the instructions (even from epix are always sorted?

Reply:
Good catch, so we can remove this